src/scripts/OSFramework/Helper/Device.ts

```diff
@@ -347,8 +347,8 @@ namespace OSFramework.Helper {
 						? sessionStorage.previewDevicesPixelRatio
 						: window.devicePixelRatio) || 1;
 				const currScreen: iphoneDetails = {
-					width: window.visualViewport.width * ratio,
-					height: window.visualViewport.height * ratio,
+					width: window.visualViewport ? window.visualViewport.width : window.innerWidth * ratio,
```

Comment:
@BenOsodrac Wouldn't it be easier to read if we had some parenthesis, like:
`(window.visualViewport ? window.visualViewport.width : window.innerWidth) * ratio`

Reply:
Made the change!